flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/TimeWindowTest.java

```diff
@@ -29,23 +29,38 @@
 public class TimeWindowTest {
     @Test
     public void testGetWindowStartWithOffset() {
-        // [0, 7), [7, 14), [14, 21)...
+        // [-21, 14), [-14, -7), [-7, 0), [0, 7), [7, 14), [14, 21)...
```

Comment:
```suggestion
        // [-21, -14), [-14, -7), [-7, 0), [0, 7), [7, 14), [14, 21)...
```

Reply:
Thanks, it was an oversight, we've fixed it